tinywallet/tinywallet/app.py

```diff
@@ -117,12 +120,12 @@ def __init__(self, qApp):
         self.loadSettings()
 
         dcrdataDB = database.KeyValueDatabase(
-            os.path.join(self.netDirectory(), "dcr.db")
+            os.path.join(self.netDirectory(), "wallet.db")
```

Comment:
This isn't the wallet database. This is the Decred blockchain database, and this filename will conflict with the wallet database.

Reply:
Whoops, sorry, reverting.